tests/bootstrap.php

```diff
@@ -0,0 +1,31 @@
+<?php
+require_once __DIR__ . '/../vendor/autoload.php';
+
+// Cleanup previously created Symfony container and logs
+$rmDir = function ($path) {
+    if (!file_exists($path) || !is_dir($path)) {
+        return;
+    }
+    $dh = new RecursiveIteratorIterator(
+        new RecursiveDirectoryIterator($path, FilesystemIterator::SKIP_DOTS | FilesystemIterator::CURRENT_AS_FILEINFO),
+        RecursiveIteratorIterator::CHILD_FIRST
+    );
+    /** @var \SplFileInfo $file */
+    foreach ($dh as $file) {
+        if (!$file->isFile() && !$file->isLink()) {
+            continue;
+        }
+        unlink($file->getPathname());
+    }
+    $dh = new RecursiveIteratorIterator(
+        new RecursiveDirectoryIterator($path, FilesystemIterator::SKIP_DOTS | FilesystemIterator::CURRENT_AS_FILEINFO),
+        RecursiveIteratorIterator::CHILD_FIRST
+    );
+    foreach ($dh as $file) {
+        rmdir($file);
+    }
+    rmdir($path);
+};
+$rmDir(__DIR__ . '/Fixtures/cache');
```

Comment:
Do we really need to cleanup those repositories ?
I need to check but it seems that in "test" mode, cache is refreshed each times.

Reply:
Yes, it is necessary, especially when testing against several different versions of Symfony on same codebase (e.g. on developer's machine). Due to change of cache directory structure in Symfony over time lack of this cleanup sometimes results into fatal errors.

Reply:
Ok ! 
Maybe we could use Filesystem component from symfony to make it more clear ?

Reply:
Yes, it is way better. Updated code to use this component